tests/commandline/test.py

```diff
@@ -219,18 +219,17 @@ def runTest():
         desc = t[1]
         expectedResult = (len(t) == 3) and 1 or 0
 
-        if useWine:
```

Comment:
How come you're removing the wine stuff?

Reply:
Wine functionality was not removed.  The boolean `useWine`  was replaced by the variable `runtimeEnvironment` set to the value of `wine`

https://github.com/nonameentername/csound/blob/e60cf81319f34e220b0ef464d8c9d5bd341bf49e/tests/commandline/CMakeLists.txt#L4